package.json

```diff
@@ -306,16 +306,18 @@
 			"npm run lint-lockfile",
 			"node ./bin/check-latest-npm.js"
 		],
+		"*.{js,json,ts,tsx,yml,yaml}": [
```

Comment:
Does anyone know how to prevent commit when formatting applies changes to modified files?

Reply:
It looks like `lint-staged` v10 adds changes to the commit automatically 🎉